fbw-a380x/src/wasm/systems/a380_systems/src/fuel/fuel_quantity_management_system.rs

```diff
@@ -309,49 +309,44 @@ impl RefuelApplication {
 
     fn lookup_trim_fuel_from_target_fuel_range(
         &mut self,
-        category_index: usize,
+        zfw_category: &HashMap<u32, Vec<u32>>,
         total_fuel_desired_rounded: u32,
         zero_fuel_weight_cg_rounded: u32,
     ) -> Mass {
```

Comment:
```suggestion
    fn lookup_trim_fuel_from_target_fuel_range(
        target_zfw_cg: &[u32],
        zfw_category: &HashMap<u32, Vec<u32>>,
        total_fuel_desired_rounded: u32,
        zero_fuel_weight_cg_rounded: u32,
    ) -> Mass {
```
Pass `target_zfw_cg` as a parameter and make function a static function.